### ISSUE TITLE
String enconding issue

### ISSUE THREAD

**Issue by florentb** [node-influx/node-influx#267]
I've got an issue while using accented characters (and possibly other special chars) in `string` fields.

Let's take this basic exemple:

```js
'use strict';

const influx = require('influx');

const influxConfig = {
  host: 'localhost',
  port: 8086,
  protocol: 'http',
  database: 'test',
  schema: [
    {
      measurement: 'test_measurement',
      fields: {
        testBoolean: influx.FieldType.BOOLEAN,
        testString: influx.FieldType.STRING,
        testInteger: influx.FieldType.INTEGER,
        testFloat: influx.FieldType.STRING,
      },
      tags: [ 'test' ]
    }
  ]
};

const influxCli = new influx.InfluxDB(influxConfig);

influxCli.writePoints([
  {
    measurement: 'test_measurement',
    fields: { testBoolean: true, testString: 'test é', testInteger: 1, testFloat: 0.1 },
    tags: { test: 'test' }
  }
], {
  precision: 'ms'
}).catch(err => {
  console.error(`Error saving data to InfluxDB! ${err.stack}`);
});
```

Note the `é` character in the testString field. 

It fails with the following error:

```
Error: A 400 Bad Request error occurred: {"error":"unable to parse 'test_measurement,test=test testBoolean=T,testFloat=\"0.1\",testInteger=1i,testString=\"test é': unbalanced quotes"}
```

InfluxDB version 1.2.2 — Node Influx version 5.0.6

**Comment by protej** [comment]
Same thing:
`
let lat_s = `${latDeg}\xB0${latMin}.${latMindec} ${data.latPole}`;
let lon_s = `${lonDeg}\xB0${lonMin}.${lonMindec} ${data.lonPole}`;
...........

          influx.writePoints([{
            measurement: 'Position',
            fields: {
              latitude: lat_s,
              longitude: lon_s
            },
            tags: {
              status: data.status
            }
          }])

`
Results in
`Error saving position to InfluxDB! Error: A 400 Bad Request error occurred: {"error":"unable to parse 'Position,status=valid latitude=\"33°49.53520\",longitude=\"151°04.6968': unbalanced quotes"}`

**Comment by connor4312** [comment]
This should be fixed in 5.0.7, which I am publishing momentarily. Thanks for opening an issue!
